python/primihub/FL/model/logistic_regression/homo_lr.py

```diff
@@ -9,15 +9,157 @@
 import pickle
 from primihub.FL.model.logistic_regression.vfl.evaluation_lr import evaluator
 
-from primihub.FL.model.logistic_regression.homo_lr_base import LRModel
```

Comment:
![Uploading image.png…]()

Reply:
Add feature selection, which can be set in feature_names in config